docs/src/user-guide/command-line-interface.md

```diff
@@ -38,80 +38,90 @@ npx eslint "lib/**"
 
 **Note:** You can also use alternative package managers such as [Yarn](https://yarnpkg.com/) or [pnpm](https://pnpm.io/) to run ESLint. Please refer to your package manager's documentation for the correct syntax.
 
+## Pass multiple values to an option
+
+Options that accept multiple values can be specified by repeating the option or with a comma-delimited list (other than [`--ignore-pattern`](#--ignore-pattern), which does not allow the second style).
+
+Examples of options that accept multiple values:
+
+```shell
+npx eslint --ext .jsx --ext .js lib/
+# OR
+npx eslint --ext .jsx,.js lib/
+```
+
 ## Options
 
-The command line utility has several options. You can view the options by running `npx eslint -h`.
+You can view all the CLI options by running `npx eslint -h`.
 
-```text
+```txt
 eslint [options] file.js [file.js] [dir]
 
 Basic configuration:
-  --no-eslintrc                   Disable use of configuration from .eslintrc.*
-  -c, --config path::String       Use this configuration, overriding .eslintrc.* config options if present
-  --env [String]                  Specify environments
-  --ext [String]                  Specify JavaScript file extensions
-  --global [String]               Define global variables
-  --parser String                 Specify the parser to be used
-  --parser-options Object         Specify parser options
-  --resolve-plugins-relative-to path::String  A folder where plugins should be resolved from, CWD by default
-
-Specifying rules and plugins:
-  --plugin [String]               Specify plugins
-  --rule Object                   Specify rules
-  --rulesdir [path::String]       Load additional rules from this directory. Deprecated: Use rules from plugins
-
-Fixing problems:
-  --fix                           Automatically fix problems
-  --fix-dry-run                   Automatically fix problems without saving the changes to the file system
-  --fix-type Array                Specify the types of fixes to apply (directive, problem, suggestion, layout)
-
-Ignoring files:
-  --ignore-path path::String      Specify path of ignore file
-  --no-ignore                     Disable use of ignore files and patterns
-  --ignore-pattern [String]       Pattern of files to ignore (in addition to those in .eslintignore)
-
-Using stdin:
-  --stdin                         Lint code provided on <STDIN> - default: false
-  --stdin-filename String         Specify filename to process STDIN as
+  --no-config-lookup             Disable look up for eslint.config.js
```

Comment:
This is the output of `-h `in flat config mode, but this document should still be about the eslintrc config mode. To switch back to the eslintrc mode, you can set `ESLINT_USE_FLAT_CONFIG=false` environment variable, and then run `npx eslint -h` again to get the help output in eslintrc config mode.